locales/en-US/app.ftl

```diff
@@ -123,7 +125,7 @@ experimentPromoSubheader = We're building next-generation features for Firefox.
 [[experimentPage]]
 isEnabledStatusMessage = {$title} is enabled.
 installErrorMessage = Uh oh. {$title} could not be enabled. Try again later.
-participantCount = <span>{$installation_count}</span> participants
+participantCount2 = {$installation_count} participants
```

Comment:
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Use_proper_plural_forms and http://projectfluent.io/fluent/guide/selectors.html

Easy example: "1 participants" is broken in English. That breaks a lot easier in other languages, where there are 4 or 5 plural forms.